security/form_login_setup.rst

```diff
@@ -137,11 +146,17 @@ a traditional HTML form that submits to ``/login``:
             <div class="alert alert-danger">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
         {% endif %}
 
+        {% if app.user %}
```

Comment:
This is not needed since the controller rendering this route redirects if there is an authenticated user.

Reply:
It comes from https://github.com/symfony/maker-bundle/blob/f7dd84691dbf399137a5caafc90f419801d09e5f/src/Resources/skeleton/authenticator/login_form.tpl.php#L12

The goal of the PR was to sync the docs with the code generated by the Maker Bundle, so if it is not necessary, it should probably be changed there first?

Reply:
Indeed :)